Scripts/ReadPDFFileV2/ReadPDFFileV2.py

```diff
@@ -213,6 +213,21 @@ def build_readpdf_entry_object(pdf_file, metadata, text, urls, images):
     return results
 
 
+def get_urls_from_binary_file(file_path):
+    """Reading from the binary pdf in the pdf_text_output_path and returns a list of the urls in the file"""
+    with open(file_path, 'rb') as file:
+        # the urls usually appear in the form: '/URI (url)'
+        urls = re.findall(r'/URI \((.*?)\)', str(file.read()))
+
+    binary_file_urls = list()
```

Comment:
How can we do union between list and set?

Reply:
A weird error was thrown when I tried to convert to set, checked this union in python console and it worked. Now I tried it again and the error is gone, but I don't think this casting is mandatory. 
Can do both ways.
Can also initiate it to be set rather than a list

Reply:
I think that set would be nicer here, so please do that